src/constants.ts

```diff
@@ -65,7 +65,9 @@ export const COINGECKO_ASSET_PLATFORMS = Object.freeze({
   [NETWORK.celo]: 'celo',
   [NETWORK.rinkeby]: 'rinkeby',
   [NETWORK.mumbai]: 'mumbai',
-  [NETWORK.avalanche]: 'avalanche'
+  [NETWORK.avalanche]: 'avalanche',
+  [NETWORK.optimism]: 'ethereum',
+  [NETWORK['optimism-kovan']]: 'ethereum',
```

Comment:
It looks like CoinGecko supports optimism as an asset platform, we should use the data from there rather than using ethereum data

https://api.coingecko.com/api/v3/asset_platforms

`optimistic-ethereum`